ilastik/applets/tracking/conservation/opConservationTracking.py

```diff
@@ -435,7 +436,7 @@ def track(self,
         trackingGraph = hypothesesGraph.toTrackingGraph()
         trackingGraph.convexifyCosts()
         model = trackingGraph.model
-        model['settings']['allowLengthOneTracks'] = False
+        model['settings']['allowLengthOneTracks'] = True
```

Comment:
We do need an option for users to filter these length-one tracks from the output, or are they filtered away by default? The reason is that having single-frame tracks popping up from time to time will confuse any analysis steps build for ilastik versions up to now. Users must be made aware of this change, or we should produce output (= images, CSV table, other output plugins) without length one tracks by default.